src/infra/modules/backend/ecs.tf

```diff
@@ -356,22 +356,23 @@ module "elb" {
 }
 
 resource "aws_cloudwatch_metric_alarm" "cpu_utilization_high" {
```

Comment:
I think the name of the terraform resource is not really appropriate.  If I am not wrong is not always cpu: it's also memory. Let's  use something more generic (such as ecs_alarms) shall we?